packages/eas-cli/src/credentials/context.ts

```diff
@@ -12,6 +12,11 @@ import * as IosGraphqlClient from './ios/api/GraphqlClient';
 import AppStoreApi from './ios/appstore/AppStoreApi';
 import { AuthenticationMode } from './ios/appstore/authenticateTypes';
 
+export type CredentialsContextProjectInfo = {
+  exp: ExpoConfig;
+  projectId: string;
```

Comment:
These are the pieces of the context that determine whether the command is running in a project or outside of a project (even though that was broken up until this series of PRs). Previously either exp was null or non-null to indicate this, but no that project ID is also required if running in a project, either these both need to be null or neither needs to be null. It seemed like the cleanest way to ensure this was to bundle the two together in a struct, but I'm open to other suggestions.

Reply:
Ah, that makes sense, did not realize that this object always refers to the current project. I'm fine with keeping current naming, so I'm leaving it up to you, but maybe more explicit naming would help e.g.
- `projectInfo` -> `currentProjectInfo`, `localProjectInfo`
- `CredentialsContextProjectInfo` -> `CredentialsCurrentProjectInfo`


For exp it was not a problem because config can only be read from a local project, but projectId potentially could identify a different project than the local one. Note that I'm not sure if with the current implementation we ever use projectId for non local project.